modules/migration-adding-cluster-to-cam.adoc

```diff
@@ -40,6 +40,10 @@ eyJhbGciOiJSUzI1NiIsImtpZCI6IiJ9.eyJpc3MiOiJrdWJlcm5ldGVzL3NlcnZpY2VhY2NvdW50Iiw
 * *Cluster name*: May contain lower-case letters (`a-z`) and numbers (`0-9`). Must not contain spaces or international characters.
 * *Url*: URL of the cluster's API server, for example, `\https://<master1.example.com>:8443`.
 * *Service account token*: String that you obtained from the source cluster.
+* *Exposed route to image registry*: Optional. You can specify a route to the image registry of your source cluster to enable direct migration for images, for example, `docker-registry-default.apps.cluster.com`.
```

Comment:
Suggestion: s/Exposed/Expose

Reply:
No. That changes the meaning. 

First, the markup indicates that this is the UI. Documentation must follow the UI, not vice versa. 

Second, the field refers to an "exposed route". It's not a procedure step. The user has exposed a route to the image registry and needs to enter the route in this field.